operator/apis/metrics/v1alpha1/common/common.go

```diff
@@ -0,0 +1,12 @@
+package common
+
+import (
+	"github.com/pkg/errors"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
let's remove those comments as those are only intended for the api struct type definitions